Tasks/AzureFileCopy/AzureUtilityGTE1.1.0.ps1

```diff
@@ -2,6 +2,31 @@
 
 Import-Module ./AzureUtilityGTE1.0.ps1 -Force
 
+function Get-AzureRMVMsInResourceGroup
```

Comment:
Can you help me what is the compat break in this function with regard to older PS module version?